scenarios/kubernetes_bazel.py

```diff
@@ -50,17 +54,80 @@ def get_version():
     with open('bazel-genfiles/version') as fp:
         return fp.read().strip()
 
+def get_affected(base, pull):
+    """Get affected packages between base sha and pull sha."""
+    diff = check_output(
+        'git', 'diff', '--name-only',
+        '--diff-filter=d', '%s...%s' % (base, pull))
+    return check_output(
+        'bazel', 'query',
+        '--noshow_progress',
+        'set(%s)' % diff).split('\n')
+
+def query(kind, packages):
+    """
+    Run a bazel query against target kind, include targets from args.
+
+    Returns a list of kind objects from bazel query.
+    """
+    return check_output(
+        'bazel', 'query',
+        '--keep_going',
+        '--noshow_progress',
+        'kind(%s, rdeps(//..., set(%s)))' % (kind, ' '.join(packages))
```

Comment:
(read this comment after the one below)

replace `//...` here with (roughly) `"set(%s)" % ' '.join(selected_packages)` where `selected_packages` is `args.build` or `args.test`.

(you may want to rename `packages` here to `changed_packages` to make this code more clear.)